easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'Sumo'
+version = '1.3.1'
+
+homepage = "https://sumo.sourceforge.net/"
+description = """Sumo is an open source, highly portable, microscopic and continuous traffic simulation package 
+ designed to handle large road networks."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+local_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
```

Comment:
@ekieffer this local parameter is not used anywhere, so can we remove it?

Reply:
Ah yes, sure ! forgot to remove it. thx